### DISCUSSION TITLE
Deployed. But not able to add custom domain.

### DISCUSSION THREAD

**Discussion by YuvarajMerugu** [vercel/community#5816] [Help]
### Summary

I deployed an app yesterday. And I added custom domain. I added A and CNAME in godaddy and it worked completely fine till today. But suddenly now i can't see my website in that domain.

Iam seeing this
This site can’t be reachedwww.eatwelll.in’s DNS address could not be found. Diagnosing the problem.
DNS_PROBE_POSSIBLE

and in the domains section it is showing that the configuration is invalid.
Please any one can tell what is the issue?

Thank you. 

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/118065209/b3e91e91-1174-4e90-bf79-12e32eca3c4f)
![image](https://github.com/vercel/community/assets/118065209/7f7f7658-0b7a-487a-ba1d-1eb12db82418)

**Comment by amyegan** [comment]
It looks like the site and domain are working correctly now. Do you still see the error?

**Reply by YuvarajMerugu** [reply]
No the error is resolved.
